db/data_migration/20160311093609_generate_content_ids_on_attachments.rb

```diff
@@ -0,0 +1,6 @@
+require 'securerandom'
+
+Attachment.where(content_id: nil).find_each do |attachment|
```

Comment:
Can that happen even when inserting them? (I mean as opposed to setting them as a default or something). 

As in `UPDATE attachments SET content_id = UUID()`